indico/web/client/styles/legacy/Conf_Basic.scss

```diff
@@ -46,10 +89,12 @@
 }
 
 .confTitleBox {
-  color: white;
+  color: var(--conf-theme-header-text);
   min-height: 90px;
   text-align: left;
-  background: #1a64a0;
+  border-top: 3px solid var(--conf-theme-header-border-top);
```

Comment:
This 3px border looks quite out of place IMHO. It was previously only set in the non-standard themes and the `standard.css` which is a bit misleading because it's the "you can download this as a starting point" css file which is never used directly by indico.